### DISCUSSION TITLE
Select discreet staves

### DISCUSSION THREAD

**Discussion by eakwarren** [musescore/MuseScore#17952] [Ideas]
@Tantacrul It would be very helpful to be able to select discreet staves. For example, in either the main score or in the Instruments panel, to hear flutes and clarinets only click on a flute stave and then via a modifier key (perhaps Alt/Option), click on the clarinet stave and _only_ those two staves are selected for playback. The oboe stave is _not_ part of the selection.

The solo buttons in the mixer can accomplish this, but it's cumbersome. Click (or F10) to show Mixer, locate the proper instrument channels (which may not be in the same view if flutes and violins are needing selection.) Select solo on each channel desired, then adjust dynamics in score, unselect solo for those channels. Repeat to compare other instrument balances, etc. It can also be done in the Instrument panel by hiding staves between the desired instruments, but again this is cumbersome.

I looked in the manual and didn't find this ability documented. Is it possible I missed it?

Thanks!

Eric

**Comment by Tantacrul** [comment]
100%. We really do need this feature. It's certainly one we know about.

@bkunda @shoogle - perhaps this could be another Community Project proposal? For sure, it needs to be designed first (probably not that hard, I'd say)

**Reply by eakwarren** [reply]
@MarcSabatella outlines the current design[ here](https://github.com/musescore/MuseScore/issues/13579#issuecomment-1587829173).
